ReactSkia/views/common/RSkTextUtils.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 1994-2021 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code isLineHeight licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#pragma once
+
+#include "ReactSkia/components/RSkComponentText.h"
+
+#include "include/core/SkFont.h"
+#include "include/core/SkPaint.h"
```

Comment:
My understanding is RSKCOmponentText.h -> RSKComponent.h -> SkCanvas.h 
this SkCanvas.h already would be including most of the skia core files. Please recheck if these header files are required again to be included

Reply:
updated